scripts/scil_score_tractogram.py

```diff
@@ -0,0 +1,376 @@
+            wpc_streamlines_count / total_count
+        final_results['wpc_bundle'] = len(
+            [x for x in wpc_streamlines_list if x])
+
+    final_results['total_streamlines'] = total_count
+    final_results["bundle_wise"] = {}
+    final_results["bundle_wise"]["true_connections"] = {}
+    final_results["bundle_wise"]["false_connections"] = {}
+    tractogram_overlap = 0.0
+
+    for i, filename in enumerate(tc_filenames):
+        current_tc_streamlines = tc_streamlines_list[i]
+        current_tc_voxels, current_tc_endpoints_voxels = get_binary_maps(
+            current_tc_streamlines, dimensions, sft, args.remove_invalid)
+
+        if args.wrong_path_as_separate:
+            current_wpc_streamlines = wpc_streamlines_list[i]
+            current_wpc_voxels, _ = get_binary_maps(
+                current_wpc_streamlines, dimensions, sft, args.remove_invalid)
+
+        tmp_dict = {}
+        tmp_dict['tc_streamlines'] = len(current_tc_streamlines)
+
+        tmp_dict['tc_dice'] = compute_dice_voxel(gt_bundle_masks[i],
+                                                 current_tc_voxels)[0]
+
+        bundle_overlap = gt_bundle_masks[i] * current_tc_voxels
+        bundle_overreach = np.zeros(dimensions)
+        bundle_overreach[np.where(
+            (gt_bundle_masks[i] == 0) & (current_tc_voxels >= 1))] = 1
+        bundle_lacking = np.zeros(dimensions)
+        bundle_lacking[np.where(
+            (gt_bundle_masks[i] == 1) & (current_tc_voxels == 0))] = 1
+
+        if args.wrong_path_as_separate:
+            tmp_dict['wpc_streamlines'] = len(current_wpc_streamlines)
+            tmp_dict['wpc_dice'] = \
+                compute_dice_voxel(gt_bundle_masks[i],
+                                   current_wpc_voxels)[0]
+            # Add wrong path to overreach
+            bundle_overreach[np.where(
+                (gt_bundle_masks[i] == 0) & (current_wpc_voxels >= 1))] = 1
+
+        tmp_dict['tc_bundle_overlap'] = np.count_nonzero(bundle_overlap)
+        tmp_dict['tc_bundle_overreach'] = \
+            np.count_nonzero(bundle_overreach)
+        tmp_dict['tc_bundle_lacking'] = np.count_nonzero(bundle_lacking)
+        tmp_dict['tc_bundle_overlap_PCT'] = \
+            tmp_dict['tc_bundle_overlap'] / \
+            (tmp_dict['tc_bundle_overlap'] +
+                tmp_dict['tc_bundle_lacking'])
+        tractogram_overlap += tmp_dict['tc_bundle_overlap_PCT']
+
+        endpoints_overlap = \
+            gt_bundle_masks[i] * current_tc_endpoints_voxels
+        endpoints_overreach = np.zeros(dimensions)
+        endpoints_overreach[np.where(
+            (gt_bundle_masks[i] == 0) &
+            (current_tc_endpoints_voxels >= 1))] = 1
+        tmp_dict['tc_endpoints_overlap'] = np.count_nonzero(
+            endpoints_overlap)
+        tmp_dict['tc_endpoints_overreach'] = np.count_nonzero(
+            endpoints_overreach)
+
+        final_results["bundle_wise"]["true_connections"][str(filename)] = \
```

Comment:
Minor: There is a mix of `"` and `'` for string here, a few line above you use `'` for all str